<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditDialog</class>
 <widget class="QDialog" name="AddEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Equipment</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="modelLabel">
       <property name="text">
        <string>Model:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="modelEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="typeLabel">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="typeComboBox">
       <item>
        <property name="text">
         <string>Excavator</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bulldozer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Crane</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Loader</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Truck</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Other</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="yearLabel">
       <property name="text">
        <string>Year:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="yearSpinBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="regNumberLabel">
       <property name="text">
        <string>Registration Number:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="regNumberEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="hoursLabel">
       <property name="text">
        <string>Operating Hours:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="hoursSpinBox"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="conditionLabel">
       <property name="text">
        <string>Condition:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="conditionComboBox">
       <item>
        <property name="text">
         <string>Excellent</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Good</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fair</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Poor</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Needs Repair</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="maintenanceLabel">
       <property name="text">
        <string>Last Maintenance:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDateEdit" name="maintenanceDateEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui> 